smalt/0.7.6/Dockerfile

```diff
@@ -0,0 +1,47 @@
+FROM continuumio/miniconda:4.7.12 as app
+
+# for easy upgrade later. ARG variables only persist during image build time
+ARG SMALT_VER="0.7.6"
+ARG SAMTOOLSVER="1.15"
+ARG BAMVER="0.0.50"
+
+# metadata
+LABEL base.image="continuumio/miniconda:4.7.12"
+LABEL dockerfile.version="1"
+LABEL software="Smalt"
+LABEL software.version="0.7.6"
+LABEL description="SMALT aligns DNA sequencing reads with a reference genome."
+LABEL website="https://www.sanger.ac.uk/tool/smalt-0/"
+LABEL license="https://www.sanger.ac.uk/tool/smalt-0/"
+LABEL maintainer1="Jill Hagey"
+LABEL maintainer.email1="jvhagey@gmail.com"
+
+# set the environment
+ENV PATH="/miniconda/bin:$PATH"
+
+RUN conda config --add channels defaults && \
+    conda config --add channels conda-forge && \
+    conda config --add channels bioconda && \
+    conda install smalt=${SMALT_VER} samtools=${SAMTOOLSVER} bambamc=${BAMVER} && \
```

Comment:
Any reason to use conda for installation instead of from source? Sometimes conda is unreliable.

Reply:
I was making several containers and one was a pain from the source so I just did them all with conda. I can redo it from the source it was more just simplicity since I saw some containers in the repo were made that way.

Reply:
Some containers do have conda installations, but we try to encourage installation from source whenever possible.

Reply:
Agreed w Erin, I prefer installing from pre-compiled binaries when available, then from source (as long as it isn't too crazy), and then conda envs when there's no other choice. Though conda is nice when the author provides an `environment.yml`. Better than relying upon a bioconda package in most cases